guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
+
+.Prerequisites
+Register a host with the created activation key. 
+For more information, see xref:Using_Activation_Keys_for_Host_Registration_{context}[Using activation keys for host registration].
+
+[NOTE]
+====
+Click the *Insecure* checkbox when registering a host.
+====
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the *New UI* button.
```

Comment:
If you have the 'Show new host overview page' setting on, you will see the new UI by default. Not sure how this should be handled in the docs.

Reply:
I added it as a note after the first step to say `If you have the *Show new host overview page* setting enabled, the new UI appears by default.`